### DISCUSSION TITLE
HA2023.8

### DISCUSSION THREAD

**Discussion by DavidMW38** [krahabb/meross_lan#308] [General]
Thank you so much for updating Meross LAN in time for 2023.8. I was one of the people that had major issues with the OS beta that stopped Meross LAN (and lots of other integrations) from working.....even after I used a full backup to go back to 2023.7.3. Don't understand how a full backup didn't fix it, but never mind lol.

I was a true beginner when I put together my HA system and wasn't happy with how I had set everything up, so with quite a few integrations not working, I decided to do a clean install and my new streamlined HA is back up and running now (on 2023.7.3, no more OS beta's for me lol). It's great to know that when I do upgrade to 2023.8 my Meross devices will work.

So, that was a really long winded way of saying.....Thank you

**Comment by PV-Joe** [comment]
HA 2023.8:
500 Internal Server Error Server got itself in trouble

I updated the final 2023.8 release last night and now my Meross Smart Plug broke.

I tried reinstalling with no luck and don‘t know how to resolve the problem.

![IMG_4059](https://github.com/krahabb/meross_lan/assets/110498562/98c021c0-c4e7-4785-8ac0-e0a389691115)

![IMG_4058](https://github.com/krahabb/meross_lan/assets/110498562/a88d5423-bc0e-4a27-948e-4040d97843e8)

Any suggestions?

**Reply by Auka84** [reply]
Hi, I have the same since I update my HA to the same version so an update of the plug in ll be necessary :( 
![Capture d’écran 2023-08-03 à 15 07 24](https://github.com/krahabb/meross_lan/assets/86434173/5a5747f1-bb67-47e3-b2ed-097352f8a645)

**Reply by krahabb** [reply]
Hello @Auka84,
check through HACS the update which would fix the 2023.8 issue: latest release is 4.3.0 and should correctly fix any

**Reply by PV-Joe** [reply]
Thanx for your reply.
Updating via HACS solved the problem 😄👍

**Reply by Auka84** [reply]
Thx a lot :). It works also for me
